StudyConnect.API/Program.cs

```diff
@@ -20,18 +20,35 @@
         }
     );
 builder.Services.AddDbContext<StudyConnectDbContext>(options =>
-    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
+{
+    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"));
+    options.EnableSensitiveDataLogging(); // Enable sensitive data logging for debugging purposes
+    options.EnableDetailedErrors();       // ausführlichere Fehlermeldungen
+    options.LogTo(Console.WriteLine, LogLevel.Trace); // Log SQL queries to the console
```

Comment:
```suggestion
{
    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"));
    options.EnableDetailedErrors();       // enable detailed error messages
    options.LogTo(Console.WriteLine, LogLevel.Debug); // Log SQL queries to the console
```
Remove detailed error messages but allowing for debugging messages should be fine.